### ISSUE TITLE
Legacy BIOS Full ROM compile from source for Braswell Chromebook Acer CB3-532

### ISSUE THREAD

**Issue by ghost** [MrChromebox/firmware#329]
Is there a way to compile an upstream (not UEFI) Full Legacy BIOS ROM from source for Braswell Chromebook (especially my Acer CB3-532)?

**Comment by MrChromebox** [comment]
absolutely, though I don't recall offhand what tweaks/config changes etc are needed for SeaBIOS to work properly on Braswell with upstream coreboot. Why are you looking for a legacy boot ROM?

**Comment by ghost** [comment]
Because I was going to try out a legacy BIOS required OS as they have no way of booting it on UEFI, but they should have it for x64._

**Comment by MrChromebox** [comment]
then I guess it's time to learn how to compile your own firmware, and likely, how to unbrick
